src/constants.ts

```diff
@@ -0,0 +1,11 @@
+import { Accessibility } from './definitions';  // Adjust the import path as necessary
+
+export const accessibilityOptions: Accessibility[] = [
```

Comment:
Why is the type duplicated?

Reply:
It is left there unintentionally. I will remove it.